.gitignore

```diff
@@ -0,0 +1,154 @@
+# FROM: https://raw.githubusercontent.com/github/gitignore/main/Python.gitignore

```

Comment:
is this a generic git ignore you throw on everything?

Reply:
github puts this out, they have boilerplate for several languages